libs/core/src/lib/card/card-footer.component.html

```diff
@@ -0,0 +1,12 @@
+<fd-toolbar size="compact" [shouldOverflow]="true" fdType="transparent" [clearBorder]="true">
+    <!-- 
+      <ng-content> approach does not work with "overflow" as we have to assign 
+      "fd-toolbar-item" directive to each fd-button to make it working.
+      The ContentChildren(FdButton) list approach with rendering in the template 
+      does not work as we can not access FdButton projected content and render it here
+      withing a loop.
+      We need to find a way to make button projected content available here to keep 
+      control over its rendering in order to make working overflow and force some options
+    -->
+    <ng-content></ng-content>
+</fd-toolbar>
```

Comment:
Would it be easier to have the application developer (user) provide the whole `fd-toolbar` as `ng-content` instead of trying to internalize here? It's not as convenient, but may be easier for us.

Reply:
Moved `toolbar` out of footer, so app developer is responsible to add it properly